### DISCUSSION TITLE
A Circle with 1km Radius

### DISCUSSION THREAD

**Discussion by lucas-vitrus** [RodrigoHamuy/react-three-map#103] [General]
Hi,

I need to draw a Circle with 1Km radius. What is the best way to convert three.js units to map units?
Is that a fixed translation? 

<img width="625" alt="image" src="https://github.com/RodrigoHamuy/react-three-map/assets/133435610/8821ccb1-f3d6-41a6-93da-f9c61eb3ad2a">

**Comment by RodrigoHamuy** [comment]
Threejs units are in meters already, so you can just add a circle with a radius of of 1000 meters (1km):
```tsx
<mesh>
  <circleGeometry args={[1000]}/>
</mesh>
```
